alphagov/verify-proxy-node
# EID-1278 - Output logs in Json format

- Bring in logstash depdency to prevent having to bring in ida-utils
- Output logs to StdOut in JSON so they can be parsed correctly by fluentd 
- Don't output to files as they will be lost when the containers are destroyed.


 **How to review**
- Run pre-commit 
- Go into the Stub Connector, Gateway and Translator pods  - kubectl exec -it '_pod-name_' bash
- Check the logs are in json format by 'kubectl logs '_pod-name_''
